src/Utils/versioninfo.jl

```diff
@@ -2,13 +2,22 @@ using Pkg
 using InteractiveUtils
 using Oceananigans.Architectures
 
+# This should be deprectated. Calls GPU() which is only
+# defined when CUDA is loaded and maps to CUDAGPU()
```

Comment:
I think at the moment only the NCDatasetExt uses this method?
do we really need it? cc @tomchor